validphys2/src/validphys/pseudodata.py

```diff
@@ -326,29 +326,54 @@ def make_level1_data(
     [CommonData(setname='NMC', ndata=204, commondataproc='DIS_NCE', nkin=3, nsys=16)]
     """
     # =============== generate experimental covariance matrix ===============#
-
     dataset_input_list = list(data.dsinputs)
-    
+
     commondata_wc = data.load_commondata_instance()
 
+    # I want to generate Lvl1 data with a varied covariance matrix while keeping the original
```

Comment:
Do we want to keep this comment? For me it is even fine, I just want to check if it is desired or not :)